seed/static/seed/js/controllers/analyses_controller.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * :copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
+ * :author
+ */
+
+angular.module('BE.seed.controller.analyses', [])
+  .controller('analyses_controller', [
+    '$scope',
+    'analyses_payload',
+    'organization_payload',
+    'users_payload',
+    'auth_payload',
+    'urls',
+    function (
+      $scope,
+      analyses_payload,
+      organization_payload,
+      users_payload,
+      auth_payload,
+      urls
+    ) {
+      $scope.org = organization_payload.organization;
+      $scope.auth = auth_payload.auth;
+      $scope.analyses = analyses_payload.analyses;
+      $scope.users = users_payload.users;
+    }
+  ])
+  .filter('get_run_duration', function() {
```

Comment:
Nice - I haven't seen this pattern in SEED, but it looks good to me! I'll again tag @axelstudios just in case we don't want to introduce a new pattern.

Reply:
Since these filters are so specific to the analysis views I'm fine with this.  For more general filters (like title-casing text) we do have a [centralized location](https://github.com/SEED-platform/seed/tree/develop/seed/static/seed/js/filters) and [filter declarations](https://github.com/SEED-platform/seed/blob/develop/seed/static/seed/js/seed.js#L98-L106)